arsceneview/src/main/java/io/github/sceneview/ar/arcore/ArSession.kt

```diff
@@ -329,4 +339,14 @@ var Config.cloudAnchorEnabled
         } else {
             Config.CloudAnchorMode.DISABLED
         }
-    }
\ No newline at end of file
+    }
+
+var Config.geospatialEnabled
+    get() = geospatialMode != Config.GeospatialMode.ENABLED
```

Comment:
I think you should write `!= Config.GeospatialMode.DISABLED` or `== Config.GeospatialMode.ENABLED`.

Reply:
Thanks! Fixed it.